### ISSUE TITLE
Indicate the name of sub-buttons on action button from index.html

### ISSUE THREAD

**Issue by Marcos-Costa** [euodeionomedeusuario/classroom#3]
Current:
![image](https://user-images.githubusercontent.com/19848508/39905723-3071e30e-54b5-11e8-8a64-4346c93a9ae1.png)
Goal:
![image](https://user-images.githubusercontent.com/19848508/39905752-549dc216-54b5-11e8-99a9-a51b2376df53.png)

**Comment by euodeionomedeusuario** [comment]
In this case, all actions are described for words and not icons.
Do you think I must add both?

![screenshot-2018-5-11 classroom](https://user-images.githubusercontent.com/20930257/39905951-683b03d2-54b6-11e8-911e-9de7e637fedc.png)

**Comment by Marcos-Costa** [comment]
Yes, as I showed up previously , the standard followed by Google Material design rule is when the button refers to a non "straight forward" icon, we possibly should describe it using few words.
The example in the picture is from Google Classroom.

**Comment by Marcos-Costa** [comment]
I updated the repository for a new version using only buttons, but as Materialize does not support using toasts side by side with floating button what I'm thinking about is implement that and also contribute to Materialize CSS oficial repository

**Comment by Marcos-Costa** [comment]
# Final result:
![image](https://user-images.githubusercontent.com/19848508/40194588-57bc2394-59e1-11e8-8527-ac08528a9c06.png)

**Comment by euodeionomedeusuario** [comment]
I saw the description of  buttons dont appear in mobile.
